src/main/java/org/folio/marccat/dao/persistence/SeeSeeAlsoReference.java

```diff
@@ -0,0 +1,40 @@
+package org.folio.marccat.dao.persistence;
+
+import org.folio.marccat.model.Subfield;
+import org.folio.marccat.util.StringText;
+
+/**
+ * @author elena
+ *
+ */
+public class SeeSeeAlsoReference extends AuthorityReferenceTag{
+	private static final String VARIANT_CODES = "weij4";
+
+
+	  public SeeSeeAlsoReference() {
+	    super();
+	  }
+
+	  @Override
+	  public StringText getStringText() {
```

Comment:
The name of this variable should be changed

Reply:
@carmentrazza Which is the new name?

Reply:
From marc21 the description of the subfield $w is  Control subfield
or or simply subfieldW, s ee you.

Reply:
I have changed it